9-regular-expressions/17-regexp-methods/article.md

```diff
@@ -1,53 +1,53 @@
-# Methods of RegExp and String
+# Métodos de RegExp e String
 
-In this article we'll cover various methods that work with regexps in-depth.
+Neste artigo, abordaremos em profundidade vários métodos que funcionam com regexps.
 
 ## str.match(regexp)
 
-The method `str.match(regexp)` finds matches for `regexp` in the string `str`.
+O método `str.macth(regexp)` pesquisa correspondências para `regexp` na string `str`.
```

Comment:
@jonnathan-ls, thanks for saving some of your time to work on this. It is very praiseworthy.

I would like to point out a little spelling mistake where it is written `str.macth` should be `str.match`.

Reply:
Thanks for the observation, I added the correction